<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pixivpy_login</class>
 <widget class="QWidget" name="pixivpy_login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>442</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	font-family: &quot;微软雅黑&quot;;
	display: block;
}
QWidget#pixivpy_login{
	background-color: transparent;
}
QLabel#background{
	background-color: #fff;
	border: 1px solid #ccc;
}
QLineEdit{
	margin: 0;
	padding: 0;
	padding-left: 10px;
	border: 1px solid #ccc;
	border-radius: 4px;
	font-size: 16px;
}
QLineEdit:placehoder-show{
	color: #ccc;
}
QPushButton{
	width: 290px;
	height: 40px;
	border-radius: 6px;
	color: #ffffff;
	font-size: 16px;
	font-weight: bold;
}
QPushButton{
	border-opacity: 200;
}
QLabel#logo{
	margin-bottom: 6px;
	background-repeat: no-repeat;	
}
QLabel#logo_text{
	color: #757c80;
	font-size: 12px;
	font-weight: 700;
}
QLabel#login_info{
	background-color: #ccc;
	border-radius: 4px;
	margin-bottom: 2px;
	color: #fff;
	font-size: 16px;
	font-weight: bold;
}
QCheckBox#remember{
	color: #757c80;
	font-size: 17px;
	font-weight: bold;
	spacing: 5px;
	
}</string>
  </property>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>290</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">#loginButton{
	background-color: #25c6ff;
	border-bottom: 3px solid #4b92ab;
	border-right: 2px solid #4b92ab;
	border-top: 1px solid #b2ebff;
	border-left: 2px solid #b2ebff;
}
#loginButton:pressed{
	border: 0;
}
#loginButton:hover{
	background-color: #75d7fa;
}</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pass_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>290</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>171</y>
     <width>290</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: darkgray;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>邮箱地址/pixiv ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource/pixiv_resource.qrc">:/login/logo.svg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="logo_text">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>363</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>让创作变得更有乐趣</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>390</y>
     <width>141</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#quitButton{
	background-color: #dc4e41;
	border-bottom: 3px solid #995751;
	border-right: 2px solid #995751;
	border-top: 1px solid #f7bdb7;
	border-left: 2px solid #f7bdb7;
}
#quitButton:pressed{
	border: 0;
}
#quitButton:hover{
	background-color: #ed8c83;
}</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="login_info">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>290</width>
     <height>42</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string>服务器异常</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="remember">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#remember:hover{
	color: #32a2e3;
}</string>
   </property>
   <property name="text">
    <string>自动登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forgetButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#forgetButton{
	color: #757c80;
	font-size: 17px;
	text-decoration: underline;
}
#forgetButton:hover{
	color: #32a2e3;
}</string>
   </property>
   <property name="text">
    <string>忘记密码?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signupButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>141</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#signupButton{
	background-color: #7eba22;
	border-bottom: 3px solid #66823c;
	border-right: 2px solid #66823c;
	border-top: 1px solid #e4f7c6;
	border-left: 2px solid #e4f7c6;
}
#signupButton:pressed{
	border: 0;
}
#signupButton:hover{
	background-color: #a4d25c;
}</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>371</width>
     <height>442</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>loginButton</zorder>
  <zorder>pass_edit</zorder>
  <zorder>id_edit</zorder>
  <zorder>logo</zorder>
  <zorder>logo_text</zorder>
  <zorder>quitButton</zorder>
  <zorder>login_info</zorder>
  <zorder>remember</zorder>
  <zorder>forgetButton</zorder>
  <zorder>signupButton</zorder>
 </widget>
 <resources>
  <include location="resource/pixiv_resource.qrc"/>
 </resources>
 <connections/>
</ui>
